helper-cli/src/main/kotlin/commands/ImportPathExcludesCommand.kt

```diff
@@ -40,7 +40,7 @@ internal class ImportPathExcludesCommand : CliktCommand(
     help = "Import path excludes by repository from a file into the given repository configuration."
 ) {
     private val pathExcludesFile by option(
-        "--path-excludes-file",
+        "--path-excludes-file", "-i",
```

Comment:
Some commands have multiple input files, do you think it is reasonable to use a '-i' option for one of them?

Reply:
I have no opinion on that, most of these commands I have never used. Is there a specific command you want me to change (or change back)?

Reply:
Anyhow, I've let the decision up to you guys so let's just leave it as is